packages/http-client/src/utils.ts

```diff
@@ -13,7 +13,7 @@ export async function fetchJson(url: string, opts: FetchOpts = {}): Promise<any>
             headers: { 'Content-Type': 'application/json' }
         })
     }
-    const res = await (await fetch(url, opts)).json()
+    const res = await fetch(url, opts).then(response => response.json())
```

Comment:
Any particular reason this was changed?

Reply:
Double wrap/unwrap of promises looks ugly and is 2 times slower than chained „then“. The change does not affect error propagation or control flow, so it is just less ugly.

Reply:
Stumbled upon it while checking the api correctness after the rename.

Reply:
Oh, I didn't know that they were slower!

Reply:
Just in case, proof: https://gist.github.com/ukstv/10cee8e47b001ffc2feebda8582cbe78